config/dependencies/argo-system/argo-events-helm-release.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: argo-events
+spec:
+  interval: 5m
+  chart:
+    spec:
+      chart: argo-events
+      version: "2.4.16"
+      sourceRef:
+        kind: HelmRepository
+        name: argo
+  releaseName: argo-events
+  install:
+    remediation:
+      retries: 30
+  upgrade:
+    remediation:
+      retries: 30
+  values:
+    # Deploy in namespaced mode - only watch resources in milo-system namespace
+    # This creates appropriate RBAC (Roles/RoleBindings instead of ClusterRoles)
+    # and limits the controller's scope to a single namespace
+    singleNamespace: true
```

Comment:
will this change in the future? just curious if there is anything outside of the milo namespace we would want to watch